lib/types/subnet_info.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Types.SubnetInfo do
+defmodule Types.AttSubnetInfo do
```

Comment:
To avoid unnecessary diffs showed here, I didn't yet changed the name of this file. Once the rest of the PR is approved I'll renaming this on as `att_subnet_info`.

Reply:
Yeah I know, but it doesn't work, locally git recognizes the rename, it's github who instead do the create/delete, I didn't dig enough but I'm not the only one ^^: 

https://github.com/orgs/community/discussions/43177#discussioncomment-5021208

Reply:
Yeah I know, the issue I saw a couple of days ago (on the previous PR) was that locally git recognized the rename, but the Github UI instead do the create/delete, I didn't dig enough but some other people had the same issue: 

https://github.com/orgs/community/discussions/43177#discussioncomment-5021208

It appears to be related to changes after the rename, or passing a certain changes threshold that Github reinterprets as a create/delete, maybe in this case it's not an issue because I already modified the file before the rename.

Reply:
Done!